laravel-app/routes/web.php

```diff
@@ -27,6 +28,9 @@
     Route::post('/logout', [HomeController::class, 'logout'])->name('logout');
     Route::get('/post', [PostController::class, 'index'])->name('post.index');
     Route::post('/post', [PostController::class, 'store'])->name('post.store');
+    Route::get('/profile', [ProfileController::class, 'index'])->name('profile.index');
```

Comment:
can you make a resource group of this??

Reply:
Changed to resource group for profile routing sir. Thank you ❤️